google/cloud/logging_v2/logger.py

```diff
@@ -206,7 +212,11 @@ def log_struct(self, info, *, client=None, **kw):
         https://cloud.google.com/logging/docs/reference/v2/rest/v2/entries/write
 
         Args:
-            info (dict): the log entry information
+            info (dict):
+                the log entry information. The dict must be serializable
+                to a Protobuf Struct (map from strings to Values, see
+                https://protobuf.dev/reference/protobuf/google.protobuf/#value),
+                otherwise the item will not be logged.
```

Comment:
it might be good to add more info to the main text too, instead of "Log a dictionary message"